index.html

```diff
@@ -813,6 +835,26 @@ <h3>Processing Model</h3>
   </p>
   <section>
     <h3>Supporting algorithms</h3>
+    <p>
+      The <dfn>reset observation window</dfn> steps given the argument |observer:PressureObserver|, are as follows:
+      <ul>
+        <li>
+          set |observer|.{{PressureObserver/[[ObservationWindow]]}} to an [=implementation-defined=] somewhat randomized integer value in
+          milliseconds, e.g., random between 300000 and 600000 (5 and 10 minutes).
+        </li>
+        <li>
+          set |observer|.{{PressureObserver/[[MaxChangesThreshold]]}} to an [=implementation-defined=] somewhat randomized integer
+          value of maximum allowed changes within the |observationWindow|.
+        </li>
+        <li>
+          set |observer|.{{PressureObserver/[[PenaltyDuration]]}} to an [=implementation-defined=] somewhat randomized integer value in milliseconds.
+        </li>
+        <li>
+          [=list/Empty=] the observer.{{PressureObserver/[[ChangesCountMap]]}} map.
+        </li>
+      </ul>
+      Run the [=reset observation window=] steps and start a timer to re-run the steps when the observer.{{PressureObserver/[[ObservationWindow]]}} time has passed.
+    </p>
```

Comment:
It s understandable,

Two suggestions maybe:
a) Instead of adding that "somewhat randomized..." for every  step, you could add that in the last line.
"To assure robustness in the observation window mitigation algorithm,
PressureObserver.observationWindow, PressureObserver.MaxChangesThreshold and PressureObserver.PenaltyDuration values must be randomized in an  [=implementation defined=] value span.

b) Or keep the text as such and emphasize the randomness again in the last sentence.
"Run the [=reset observation window=] step with new  somewhat randomized values and start ....